src/Sylius/Bundle/CoreBundle/Fixture/Factory/AddressExampleFactory.php

```diff
@@ -143,8 +140,6 @@ public function create(array $options = [])
             $address->setProvinceName($options['province_name']);
         }
 
-        $options['customer']->addAddress($address);
```

Comment:
Removing the feature to assign address to a customer just to make use of this factory in tests is not good... 😕